server/base/src/main/java/org/apache/accumulo/server/util/ZooZap.java

```diff
@@ -90,6 +90,19 @@ public static void main(String[] args) throws Exception {
 
   @Override
   public void execute(String[] args) throws Exception {
+    try {
+      var siteConf = SiteConfiguration.auto();
+      // Login as the server on secure HDFS
+      if (siteConf.getBoolean(Property.INSTANCE_RPC_SASL_ENABLED)) {
+        SecurityUtil.serverLogin(siteConf);
+      }
+      zap(siteConf, args);
```

Comment:
Was the change in this file only a refactoring to put the code in a the try block in a method?  Seems like no code changes were made besides that but was not sure.

Reply:
Looking at the diff ignoring whitespace changes makes this easier to see. It looks like the only thing that changed was that the jcommander option parsing got moved into the try block. Otherwise, I don't think there's any substantive change in this part.